src/main/java/com/altenheim/kalender/controller/logicController/IOController.java

```diff
@@ -0,0 +1,187 @@
+package com.altenheim.kalender.controller.logicController;
+
+import java.io.File;
+import com.altenheim.kalender.controller.viewController.CalendarViewOverride;
+import com.altenheim.kalender.interfaces.IAppointmentEntryFactory;
+import com.altenheim.kalender.interfaces.IIOController;
+import com.altenheim.kalender.models.ContactModel;
+import com.altenheim.kalender.models.MailTemplateModel;
+import com.altenheim.kalender.models.SettingsModel;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.GregorianCalendar;
+import java.util.List;
+import com.calendarfx.model.CalendarSource;
+import com.calendarfx.model.Entry;
+import net.fortuna.ical4j.data.*;
+import net.fortuna.ical4j.model.property.*;
+import net.fortuna.ical4j.model.Calendar;
+import net.fortuna.ical4j.model.DateTime;
+import net.fortuna.ical4j.model.Property;
+import net.fortuna.ical4j.model.component.VEvent;
+import net.fortuna.ical4j.util.RandomUidGenerator;
+import net.fortuna.ical4j.validate.ValidationException;
+
+public class IOController implements IIOController
+{
+
```

Comment:
In dieser Klasse (von denen einige andere erben) werden alle Operationen fürs Schreiben und Lesen vom und auf die Festplatte verwaltet. Die Schreib und Lesemethoden für den Kalender habe ich mal aus eurem Import/Export Branch übernommen, da sie IO Zugriffe machen müssen wir überlegen ob Import-Export die Klasse erbt oder wir das nochmal splitten.